### PR TITLE
Contact Us Form Visuals

### PR DESCRIPTION
**Pull request by n-mathis**
Links
-----
Task: Change Contact Us form to match wireframes and not use Google Forms

Related PRs: server/contact-us-server working on the server side of the contact forms

Description
-----------
- connected to the server side of thing with sendContact
- check the server PR for more info

Developer self-review checklist
-------------------------------
- [x] Task's requirements have been fully addressed
- [x] PR link has been posted in the task's comments
- [x] Potentially confusing code has been explained with comments
- [x] No warnings or errors have been introduced; all known error cases have been handled
- [ ] Any appropriate documentation (within the code, README.md, docs, etc) has been updated
- [x] There are no new spelling/grammar mistakes in the UI, code, or documentation
- [x] Branch has been deployed to staging, and all edge cases have been manually tested
- [x] Task and PR have been updated to show that this is ready for review

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,6 +6,7 @@ test/unit/coverage
 test/e2e/reports
 selenium-debug.log
 *.swp
+sendgrid.env
```
Comment (jkeat) [inline .gitignore]: is this necessary?

**Comment by jkeat** [comment]
Looks good for the most part! See my questions above ^ about the `.gitignore` addition.

Also, there are a few styling concerns I have around mobile, and around viewing it when signed in vs. signed out. I think some of these issues will be simpler to solve once some of @moody 's PRs are merged in.

File: .gitignore (hunk 1 above)
Comment (n-mathis) [reply .gitignore]: I don't think so. I don't think that file is even needed because it was just for setting the API key which is already set in the config file.
